code/API_definitions/edge-discovery-service.yaml

```diff
@@ -11,14 +11,198 @@ info:
   description: |
     # Introduction
     ---
-    Use the Edge Discovery API for
-    discovery of the most optimal edge cloud zone for a specific user as well as specific application instance the user is trying to connect to.
+    The Edge Discovery API is designed to help users discover the most optimal
+    edge cloud zones for their applications. This API allows users to find the
+    best regions and zones for their edge cloud applications, discover optimal
+    edge cloud zones for deployed applications, and find the best application
+    endpoints for clients to connect to.
```

Comment:
Need to refer to edge cloud zone instead of zone for consistency purpose as this is the formal term to indicate edge resources as I think.

Reply:
resolved